changelog/10077.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core: Fix client.Clone() to include the address
```

Comment:
Does this mean that changelog entries are sort of "staged", and doesn't really appear on `CHANGELOG.md` until right before release, when `changelog-build` is ran?

Reply:
Yes, that's correct. I plan on adding an update to the "next" section of the changelog on every minor release so that periodic updates do show up there.